content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md

```diff
@@ -54,8 +54,9 @@ busybox   1/1       Running   0          <some-time>
 {{< codenew file="admin/dns/dnsutils.yaml" >}}
 
 {{< note >}}
-此示例在 `default` 命名空间创建 pod。 服务的 DNS 名字解析取决于 pod 的命名空间。 详细信息请查阅
-[服务和 Pod 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names)。
+此示例在 `default` 名字空间创建 pod。
+服务的 DNS 名字解析取决于 pod 的名字空间。
+详细信息请查阅[服务和 Pod 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names)。
```

Comment:
```suggestion
详细信息请查阅 [Pod 与 Service 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names)。
```
Sometimes, if the link name is different from the linked title, people may be doubted  whether it opens a wrong page.
https://kubernetes.io/zh-cn/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names